src/readme.txt

```diff
@@ -4,10 +4,10 @@ Donate link:       https://wordpressfoundation.org
 Tags:              community, group, members, network, self-hosted
 License:           GNU General Public License v2 or later
 License URI:       https://www.gnu.org/licenses/gpl-2.0.html
-Requires PHP:      5.6
-Requires at least: 6.1
+Requires PHP:      7.0
+Requires at least: 6.4
 Tested up to:      6.6
-Stable tag:        14.0.0
+Stable tag:        15.0.0-alpha
```

Comment:
```suggestion
Stable tag:        14.0.0
```
Please do not change stable tag. This only happens when building a new minor/major version. See https://github.com/buddypress/buddypress/blob/master/docs/contributor/project/release/build-checklist.md#version-bumps-in-dev

Reply:
Ops! Good catch!